crypto/nrf_oberon/include/ocrypto_ecdsa_p256.h

```diff
@@ -20,83 +21,100 @@ extern "C" {
 #include <stddef.h>
 #include <stdint.h>
 
+
 /**
- * ECDSA P-256 signature key pair generation.
+ * ECDSA P-256 public key generation.
+ *
+ * Given a secret key @p sk the corresponding public key is computed and put
+ * into @p pk.
  *
- * @param[out]  pk  Generated public key.
- * @param[out]  sk  Private key.
+ * @param[out] pk Generated public key.
+ * @param      sk Secret key. Must be pre-filled with random data.
  *
- * @retval 0 If @p sk is a legal secret key.
+ * @retval 0  If @p sk is a valid secret key.
+ * @retval -1 Otherwise.
  */
 int ocrypto_ecdsa_p256_public_key(
     uint8_t pk[64],
     const uint8_t sk[32]);
 
-
 /**
  * ECDSA P-256 signature generation.
  *
- * @param[out]  sig     Generated signature.
- * @param       m       Input message.
- * @param       mlen    Length of the message.
- * @param       sk      Secret key.
- * @param       ek      Session key.
+ * The message @p m is signed using the secret key @p sk and the ephemeral
+ * session key @p ek. The signature is put into @p sig.
  *
- * @retval 0 If @p ek is a valid session key.
+ * @param[out] sig  Generated signature.
+ * @param      m    Input message.
+ * @param      mlen Length of @p m.
+ * @param      sk   Secret key.
+ * @param      ek   Session key.
```

Comment:
Should we namedrop "ephemeral" here to give some context to the abbreviation? Makes the code easier to read when you know the abbreviations IMO. The term 'ephemeral session key' is used elsewhere
```suggestion
 * @param      ek   Ephemeral session key.
```

Reply:
Agreed. Will fix